config/samples/backends/bases/iscsid/iscsid.yaml

```diff
@@ -12,4 +12,5 @@ spec:
     systemd:
       units:
       - enabled: true
-        name: iscsid.service
\ No newline at end of file
+        name: iscsid.service
+
```

Comment:
I think you added 2 newlines instead of just 1. L15 needs a newline, but L16 should be deleted.